src/main/scala/org/constellation/p2p/Download.scala

```diff
@@ -266,6 +263,7 @@ class DownloadProcess[F[_]: Concurrent: Timer: Clock](
     for {
       _ <- setSnapshot(snapshot)
       _ <- acceptSnapshotCacheData(snapshot)
+      _ <- storeSnapshotInfo
```

Comment:
Just my guess that we should store `SnapshotInfo` after redownloading from others as well. Is it correct @marcinwadon?